app/src/main/java/com/nextcloud/talk/utils/ApiUtils.java

```diff
@@ -37,6 +37,8 @@
 import okhttp3.Credentials;
 
 public class ApiUtils {
+    public static final int APIv3 = 3;
+    public static final int APIv4 = 4;
```

Comment:
Had to add configs to not get a warning about magic numbers (only -1 to 2 are okay, afterwards ktlint throws an error)